### PR TITLE
Add metadata to pages and site. Fix broken links on github page README

### PR DESCRIPTION
**Pull request by fluffyemily**

### CODE REVIEW

**Comment by fluffyemily** [comment]
@agi90 can you just review the metadata tags that I've added and let me know if you think there should be any others?

File: docs/geckoview-quick-start.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 ---
 layout: default
 title: Get Started with GeckoView
-nav_order: 2
+nav_order: 2 
+summary: How to use GeckoView in your Android app.
+tags: [GeckoView,Gecko,mozilla,android,WebView,mobile,mozilla-central,setup,quick start]
```
Comment (agi90) [inline docs/geckoview-quick-start.md:6]: FYI Google says it doesn't look at tags at all. This document is helpful: https://support.google.com/webmasters/answer/79812?hl=en

do you know how does the `summary` show up in the html? is it `<meta name="summary">`?

**Review by agi90 — APPROVED** [review]

File: docs/geckoview-quick-start.md (hunk 1 above)
Comment (fluffyemily) [reply docs/geckoview-quick-start.md:6]: I've also updated the head HTML template to add these meta tags. `summary` is displayed as a `description` meta tag.

Comment (agi90) [reply docs/geckoview-quick-start.md:6]: sounds good!

**Pull request merged**